emails/views.py

```diff
@@ -336,11 +341,41 @@ def _sns_message(message_json):
     return ses_relay_email(from_address, relay_address, subject, message_body)
 
 
+def _get_attachment_metrics(part):
+    incr_if_enabled('email_with_attachment', 1)
+    fn = part.get_filename()
```

Comment:
Could consolidate some more calls into local vars.
```suggestion
    incr_if_enabled('email_with_attachment', 1)
    fn = part.get_filename()
    ct = part.get_content_type()
    payload = part.get_payload(decode=True)
    payload_size = len(payload)
```